app/views/RegisterView.tsx

```diff
@@ -99,8 +101,8 @@ class RegisterView extends React.Component<IProps, any> {
 	}
 
 	login = () => {
-		const { navigation, Site_Name } = this.props;
-		navigation.navigate('LoginView', { title: Site_Name });
+		const { navigation, Site_Url } = this.props;
+		navigation.navigate('LoginView', { title: new parse(Site_Url).hostname });
```

Comment:
You've replaced Site_Name usage, so you need to remove the prop as well.

Reply:
Done!